schutzbot/manifest_tests.sh

```diff
@@ -1,10 +1,37 @@
 #!/bin/bash
+function nvrGreaterOrEqual {
+    local rpm_name=$1
+    local min_version=$2
+
+    set +e
+
+    rpm_version=$(rpm -q --qf "%{version}" "${rpm_name}")
+    rpmdev-vercmp "${rpm_version}" "${min_version}" 1>&2
+    if [ "$?" != "12" ]; then
+        # 0 - rpm_version == min_version
+        # 11 - rpm_version > min_version
+        # 12 - rpm_version < min_version
+        echo "DEBUG: ${rpm_version} >= ${min_version}" 1>&2
+        set -e
+        return
+    fi
+
+    set -e
+    false
+}
+
 set -euxo pipefail
 # Get OS details.
 source /etc/os-release
 ARCH=$(uname -m)
-DISTRO_CODE="${DISTRO_CODE:-${ID}-${VERSION_ID//./}}"
-
+# Since rhel-9.3 and rhel-8.9 the manifests are stored with dotted notation (rhel-9.3 instead of rhel-93, hence
+# adding this to handle the new notation and ensure backwards compatibility
+COMPARE=$(nvrGreaterOrEqual "rhel" "8.9")
```

Comment:
This isn't going to work!

This function queries an rpm package and compares it with the desired version. So this should be something like 
`nvrGreaterOrEqual "osbuild-composer" "100"` - where `100` is the actual version number where the repositories naming changed. I'm not sure if it was 100 exactly.

Reply:
Using osbuild-composer release [100](https://github.com/osbuild/osbuild-composer/releases/tag/v100) instead which includes [this PR](https://github.com/osbuild/osbuild-composer/pull/3887) for the addition of dot separator between major and minor version.